### ISSUE TITLE
Chat Needs Store Name 'Tooltip'

### ISSUE THREAD

**Issue by jashot7** [OpenBazaar/openbazaar-desktop#772]
![desktop_needs_storename_chat_tooltip](https://user-images.githubusercontent.com/392798/30078700-1fc77338-924c-11e7-8d6b-3844311b6df6.gif)

**Comment by morebrownies** [comment]
+1 

I suggested this way back in 1.0, but I believe it was blocked due to complexity. Hopefully it is easier in 2.0 👍

**Comment by jjeffryes** [comment]
I think this will run into similar issues with the overflow being hidden. We'll have to see if there's a clever way around it.

**Comment by rmisio** [comment]
> I think this will run into similar issues with the overflow being hidden. We'll have to see if there's a clever way around it.

yup, that was the complexity in v1 that is not "easier in 2.0" :(
